packages/web/components/Editor/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import { Controller } from 'react-hook-form';
+
+const Editor = ({ config, form, name, disabled }) => {
```

Comment:
This is a wrapper so we can use CKEditor with RHF.

We can improve it functionalities over the time

Reply:
Great implementation!

Reply:
Awesome! 💯